src/test/java/eu/europeana/metis/sandbox/dto/report/ProgressInfoDtoTest.java

```diff
@@ -133,6 +133,6 @@ private static ProgressInfoDto getTestCompletedInfoDto() {
             Language.HR,
             Country.CROATIA,
             false,
-            false));
+            false), "");
   }
```

Comment:
my suggestion is to add the following unit test:
```
 @NotNull
  private static ProgressInfoDto getTestErroTypeInfoDto() {
    return new ProgressInfoDto("http://metis-sandbox",
        5L,
        5L,
        getProgressByStepDtoList(5, 5),
        new DatasetInfoDto("datasetId",
            "datasetName",
            LocalDateTime.parse("2022-03-14T22:50:22"),
            Language.HR,
            Country.CROATIA,
            false,
            false), "Error");
  }
  
  @Test
  void getErrorType() {
    progressInfoDto = getTestErroTypeInfoDto();
    assertEquals( "Error", progressInfoDto.getErrorType());
    assertEquals( "", progressInfoDto.getPortalPublishUrl());
  }
```

Reply:
Added